### PR TITLE
DOCS-3606, DOCS-4468 Error Tracking Index Pages Doc Edit

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Adds information about the Open, Ignored, and All Issues views and updates the screenshots in the index pages to include the three views. 

### Motivation

DOCS-4468

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/logs/error_tracking/_index.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ further_reading:
 
 It is critical for your system’s health to consistently monitor the errors collected by Datadog. When there are many individual error events, it becomes hard to prioritize errors for troubleshooting.
```
Comment (alai97) [inline content/en/logs/error_tracking/_index.md:24]:
```suggestion
It is critical for your system’s health to consistently monitor the errors collected by Datadog. When there are many individual error events, it becomes hard to prioritize errors for troubleshooting. By tracking, triaging, and debugging logs, you can minimize the impact of fatal errors on your browser, mobile, and backend services.
```

File: content/en/real_user_monitoring/error_tracking/_index.md [hunk 2]
```diff
@@ -12,18 +12,23 @@ further_reading:
...
+- link: "/monitors/types/error_tracking/"
+  tag: "Documentation"
+  text: "Create an Error Tracking monitor"
 ---
 
 ## Overview
 
 It is critical for your system's health to consistently monitor the errors collected by Datadog. When there are many individual error events, it becomes hard to prioritize errors for troubleshooting. By tracking, triaging, and debugging crashes, you can minimize the impact of fatal errors on your web and mobile applications' user experience.
 
+Once you have setup RUM for **Web and Mobile Apps** error tracking, the issue list populates with cards. You can view open, ignored, or all issues, sort issues by volume or age, and filter issues by all custom and default facets on your web and mobile applications.
```
Comment (maycmlee) [inline content/en/real_user_monitoring/error_tracking/_index.md:30]:
```suggestion
Once you have set up RUM for **Web and Mobile Apps** error tracking, the issue list populates with cards. You can view open, ignored, or all issues, sort issues by volume or age, and filter issues by all custom and default facets on your web and mobile applications.
```

File: content/en/tracing/error_tracking/_index.md [hunk 3]
```diff
@@ -6,18 +6,23 @@ further_reading:
...
   tag: "Documentation"
-  text: "Learn about the Trace Explorer"
+  text: "Create an Error Tracking monitor"
 ---
 
 ## Overview
 
 It is critical for your system's health to consistently monitor the errors collected by Datadog. When there are many individual error events, it becomes hard to prioritize errors for troubleshooting. By tracking, triaging, and debugging stack traces, you can minimize the impact of fatal errors on your backend services.
 
+Once you have setup APM for **Backend Services** error tracking, the issue list populates with cards. You can view open, ignored, or all issues, sort issues by volume or age, and filter issues by all custom and default facets on your backend services.
```
Comment (maycmlee) [inline content/en/tracing/error_tracking/_index.md:24]:
```suggestion
Once you have set up APM for **Backend Services** error tracking, the issue list populates with cards. You can view open, ignored, or all issues, sort issues by volume or age, and filter issues by all custom and default facets on your backend services.
```

File: content/en/logs/error_tracking/_index.md [hunk 4]
```diff
@@ -21,9 +24,9 @@ further_reading:
 
 ## Overview
 
-It is critical for your system’s health to consistently monitor the errors collected by Datadog. When there are many individual error events, it becomes hard to prioritize errors for troubleshooting. 
+It is critical for your system’s health to consistently monitor the errors collected by Datadog. When there are many individual error events, it becomes hard to prioritize errors for troubleshooting. By tracking, triaging, and debugging logs, you can minimize the impact of fatal errors on your browser, mobile, and backend services.
 
-Once you have setup logs for **Browser and Mobile** or **Backend** error tracking, the issue list populates with cards. You can sort by volume or age, and filter by all custom and default facets on your logs.
+Once you have setup logs for **Browser and Mobile** or **Backend** error tracking, the issue list populates with cards. You can view open, ignored, or all issues, sort issues by volume or age, and filter issues by all custom and default facets on your logs.
```
Comment (maycmlee) [inline content/en/logs/error_tracking/_index.md:29]:
```suggestion
Once you have set up logs for **Browser and Mobile** or **Backend** error tracking, the issue list populates with cards. You can view open, ignored, or all issues, sort issues by volume or age, and filter issues by all custom and default facets on your logs.
```

**Review by maycmlee — APPROVED** [review]
Just a couple of small edits, but approving! You might want to rename the images so you don't run into the caching issue: https://datadoghq.atlassian.net/wiki/spaces/docs4docs/pages/1985314888/Screenshots+and+Videos#Update-Image-Repository

**Pull request merged**